tests/FieldDescription/FilterTypeGuesserTest.php

```diff
@@ -130,6 +130,8 @@ public function testGuessTypeNoAssociation(string $type, string $resultType, int
     }
 
     /**
+     * @psalm-suppress DeprecatedConstant
```

Comment:
maybe here we can only test the non deprecated constants, so on next major we dont have to change it, and we avoid a supress

Reply:
IMO it's good to test deprecated constants to see they work find and prevent to remove them accidentally, it's deprecated by `doctrine/mongodb-odm` so we do't know when we are going to remove them.

I would add these issues to `psalm.xml`, but when specifying the `DeprecatedConstant` issue, you have to say in which file and we could add more deprecated constants that have nothing to do with these ones and they would be suppressed too.